charts/bindplane/templates/bindplane.yaml

```diff
@@ -247,9 +247,12 @@ spec:
             {{- if eq .Values.auth.type "auth0" }}
             - name: BINDPLANE_AUTH_TYPE
               value: auth0
-            {{- if len .Values.extraEnv }}
-            {{- toYaml .Values.extraEnv | nindent 12 }}
-            {{- end }}
```

Comment:
Lets keep this but move it to the bottom of the `env` section, that way it can still be used.